cli/build.rs

```diff
@@ -133,7 +133,11 @@ mod ts {
       // Deno built-in type libraries
       "decorators",
       "decorators.legacy",
-      "es5",
+      "deno_webgpu",
```

Comment:
Why is this one added here?

Reply:
Oh, whoops. I did a read_dir of the tsc/dts folder